ssg/oval.py

```diff
@@ -97,7 +98,7 @@ def applicable_platforms(oval_file):
 
     platforms = []
     header = oval_generated_header("applicable_platforms", "5.11", "0.0.1")
-    body = read_ovaldefgroup_file(oval_file)
+    body = process_file(oval_file, {})
```

Comment:
To clarify why times to run this script have increased...

We're now processing every single OVAL file with JINJA macros (no cache) before trying to parse the `affected/platforms` tag. It's a big hit relative to the script (iirc 5s -> 30s), but it is still much quicker than a full build of all the content. At some point in the future, this could be dropped if someone wants to write a dirty line-based XML parser, but that's a bit more work.

Reply:
It's necessary in the general case because you could theoretically use jinja code to emit platform elements.

Reply:
But in that case, you'd need to process the OVAL with each of the platform YAML files and detect if the applicable platform changes with each call... Because you could use that same JINJA and condition on the product variable in the `product.yml` file... So it'd get really messy and I'd recommend not doing that.

Reply:
Yeah, I see your point. Probably best to say that it's not supported because it's incredibly difficult to support.

Reply:
And we'd need to support it in the fix content and the YAML files as well... Which gets awfully messy awfully quickly.